apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -268,6 +269,30 @@ const BookingPage = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const getBookingRedirectExtraParams = (booking: any) => {
+    if (!eventType.metadata?.passExtraParamsToRedirectUrl) {
+      return {};
+    }
+
+    const objectQueryParamKeys = ["customInputs", "responses", "metadata", "user", "attendees"];
+    const redirectQueryParamKeys = [
+      "title",
+      "description",
+      "startTime",
+      "endTime",
+      "location",
+      ...objectQueryParamKeys,
+    ];
+
+    const params = _.pick(booking, redirectQueryParamKeys);
```

Comment:
Lets try and not import lodash - especially in our booking page. 

I havent tested but something like this should replicate the functionality 

```suggestion
    const params = Object.keys(booking).filter(key => redirectQueryParamKeys.includes(key)).reduce((obj, key) => Object.assign(obj, { [key]: booking[key] }), {});
```

Reply:
thank you for the suggestion. added this snippet to my branch and removed lodash import from the top of the file